ports/voidstranger/port.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "voidstranger.zip",
+  "items": [
+    "Void Stranger.sh",
+    "voidstranger/"
+  ],
+  "items_opt": null,
+  "attr": {
+    "title": "Void Stranger",
+    "porter": [
+      "nate_____", "gooeyPhantasm"
+    ],
+    "desc": "Void Stranger is a 2D sokoban-style puzzle game where every step counts. Descend into the forgotten labyrinth teeming with fiendish foes and traps that defy reason. A swift defeat lurks at every corner for those who neither study their surroundings nor think their moves through with care.\nTrust you wits and slowly, surely, you will conquer the mysteries before you.\n\nYou do remember why you are here, right? Or have you lost something? Something very important to you.",
+    "desc_md": null,
+    "inst": "Purchase game via https://store.steampowered.com/app/2121980/Void_Stranger/ -Currently supporting v1.1.1 off Steam and Itch.io -Place game .png, .dat, .csv, and .win files in the \"gamedata\" folder. See readme for more detail.",
+    "inst_md": null,
```

Comment:
Inst_md field is for use on the website so supports Hyperlinks etc. So use this field for making the text nice on the website and the other one for "sinple" on device text